opendbc/safety/modes/rivian.h

```diff
@@ -43,9 +38,9 @@ static uint32_t rivian_compute_checksum(const CANPacket_t *msg) {
   uint8_t chksum = 0;
   if (msg->addr == 0x208U) {
     chksum = _rivian_compute_checksum(msg, 0x1D, 0xB1);
-  } else if (msg->addr == 0x150U) {
-    chksum = _rivian_compute_checksum(msg, 0x1D, 0x9A);
   } else {
+    // msg->addr == 0x150U
+    chksum = _rivian_compute_checksum(msg, 0x1D, 0x9A);
```

Comment:
we need to be careful to not let the tests make the actual code less readable. this is one of those cases; can you rewrite it in a better way?